packages/dx-scheduler-core/src/plugins/all-day-panel/helpers.js

```diff
@@ -1,61 +1,18 @@
 import moment from 'moment';
 
-const CELL_GAP = 0.32;
-const CELL_BOUND_OFFSET_PX = 2;
-
 export const allDayPredicate = appointment => (
   appointment.end.diff(appointment.start, 'hours') > 23
   || !!appointment.allDay
 );
 
-const getCellRect = (date, dayScale, cellElements, takePrev) => {
+export const getAllDayCellByDate = (viewCellsData, date, takePrev) => {
```

Comment:
The `getAllDayCellByDate` name sounds strange. I assume the function returns a cell, however it returns a cell index.

Reply:
Can we rename this to `getAllDayCellIndexByDate` ?

Reply:
I think so.